src/main/java/uk/gov/companieshouse/authcodenotification/service/PrivateDataRetrievalService.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.authcodenotification.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.update.OverseasEntityDataApi;
+import uk.gov.companieshouse.authcodenotification.client.ApiClientService;
+import uk.gov.companieshouse.authcodenotification.exception.ServiceException;
+import uk.gov.companieshouse.authcodenotification.utils.ApiLogger;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+import java.io.IOException;
+
+@Service
+public class PrivateDataRetrievalService {
+
+    private static final String OVERSEAS_ENTITY_URI_SECTION = "/overseas-entity/";
```

Comment:
Suggest we making this a 'formattable' string (with %s in for company number) and use the string formatter to populate it, rather than do all the string concatenation.

Reply:
yes, can remove the constant and just use /overseas-entity/ in the string.format.

Reply:
Kept the constant but made it formatable.